bpfd/src/lib.rs

```diff
@@ -141,7 +141,15 @@ pub async fn serve(config: Config, static_programs: Vec<StaticPrograms>) -> anyh
                             _ => proceed_on,
                         }
                     } else {
-                        proceed_on
+                        // FIXME: when proceed-on is supported for TC programs just return: proceed_on
+                        match program_type {
+                            command::ProgramType::Xdp => proceed_on,
+                            command::ProgramType::Tc => {
+                                warn!("proceed-on config not supported yet for TC and my have unintended behavior");
```

Comment:
```suggestion
                                warn!("proceed-on config not supported yet for TC and may have unintended behavior");
```

Reply:
> i.e ProceedOn::TX == 3 and TC_ACT_PIPE == 3 so the printing of proceed on get's all wacky

I don't think it's an issue right now because we don't seem to use it for tc programs, but that will all need to be sorted out when I handle issue #299.

E.g., we don't print out the ProceedOn config for TC now:
```
$ bpfctl list
 UUID                                  Type  Name       Location                                                                       Metadata                                                                                            
 c600bb0a-7879-4224-8c26-39c1cee177f0  tc    drop_icmp  file:///home/afredette/bpf/drop-icmp.o                                         { "priority": 50, "iface": "vboxnet1", "position": 2, direction: INGRESS }                          
 fd537f1a-d127-4f15-b1f8-4e39757cd2f4  tc    accept     file:///home/afredette/bpf/accept-all.o                                        { "priority": 40, "iface": "vboxnet1", "position": 0, direction: INGRESS }                          
 2da6b733-fef5-4078-9ed6-9f6dbecdbf12  tc    drop_icmp  file:///home/afredette/bpf/drop-icmp.o                                         { "priority": 50, "iface": "vboxnet1", "position": 3, direction: INGRESS }                          
 687dd0b3-fb5d-4175-8dc2-47c33f54168d  tc    accept     file:///home/afredette/bpf/accept-all.o                                        { "priority": 40, "iface": "vboxnet1", "position": 1, direction: INGRESS }                          
 f20a93d2-cc2f-414d-b0f8-35ae025fb937  xdp   xdp        file:///home/afredette/projects/xdp-tutorial/basic01-xdp-pass/xdp_pass_kern.o  { "priority": 30, "iface": "vboxnet1", "position": 1, "proceed_on": ["dispatcher_return"] }         
 a4a923aa-2000-4459-b374-a50cb13d1f4e  xdp   xdp        file:///home/afredette/projects/xdp-tutorial/basic01-xdp-pass/xdp_pass_kern.o  { "priority": 30, "iface": "vboxnet1", "position": 0, "proceed_on": ["pass", "dispatcher_return"] } 
```

Reply:
This was a response to your comments below.  

I'll fix the type you pointed out above.